GrandCommonDivisor.Core/Utils/GCDUtils.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GrandCommonDivisor.Core.models;
+using System;
+
+namespace GrandCommonDivisor.Core
+{
+    public static class GCDUtils
+    {
+        public static NumberParseResult CheckNumber(string number)
+        {
+
+            if (!int.TryParse(number, out var num))
+            {
+                var error = "Please try number again!";
+                return new NumberParseResult(error);
+            }
+            else
```

Comment:
You don't need that "else", since you are returning from "if" before that

Reply:
Fixed.